packages/interface/src/components/preferences-modal-tabs/style.scss

```diff
@@ -24,6 +24,12 @@ $vertical-tabs-width: 160px;
 
 			&:focus:not(:disabled) {
 				box-shadow: inset 0 0 0 var(--wp-admin-border-width-focus) var(--wp-admin-theme-color);
+				// Windows high contrast mode.
+				outline: 2px solid transparent;
```

Comment:
This style is required for high contrast mode.


https://user-images.githubusercontent.com/54422211/224363315-cf860051-352b-4335-aaec-acf43c0b9621.mp4